client/main.lua

```diff
@@ -148,58 +148,75 @@ function OpenBossMenu(groupType)
+                    icon = 'fa-solid fa-right-to-bracket',
+                    label = groupType == 'gang' and Lang:t('menu.gang_menu') or Lang:t('menu.boss_menu'),
+                    groups = groupName,
+                    onSelect = function()
+                        OpenBossMenu(groupType)
+                    end
                 }
-            })
-        end
+            }
+        })
     else
-        for groups, group in pairs(sharedConfig.menus) do
-            lib.zones.box({
-                coords = group.coords,
-                rotation = group.rotation,
-                size = group.size,
-                debug = config.debugPoly,
-                onEnter = function()
-                    if groups == QBX.PlayerData[group.group].name and QBX.PlayerData[group.group].isboss then
-                        lib.showTextUI(group.group == 'gang' and Lang:t('menu.gang_management') or Lang:t('menu.boss_management'))
-                    end
-                end,
-                onExit = function()
-                    lib.hideTextUI()
-                end,
-                inside = function()
-                    if IsControlJustPressed(0, 51) then -- E
-                        if groups == QBX.PlayerData[group.group].name and QBX.PlayerData[group.group].isboss then
-                            OpenBossMenu(group.group)
-                            lib.hideTextUI()
-                        end
+        lib.zones.box({
+            coords = coords,
+            rotation = rotation,
+            size = size,
+            debug = config.debugPoly,
+            onEnter = function()
+                if groupName == QBX.PlayerData[groupType].name and QBX.PlayerData[groupType].isboss then
+                    lib.showTextUI(groupType == 'gang' and Lang:t('menu.gang_management') or Lang:t('menu.boss_management'))
+                end
+            end,
+            onExit = function()
+                lib.hideTextUI()
+            end,
+            inside = function()
+                if IsControlJustPressed(0, 51) then -- E
+                    if groupName == QBX.PlayerData[groupType].name and QBX.PlayerData[groupType].isboss then
+                        OpenBossMenu(groupType)
+                        lib.hideTextUI()
                     end
                 end
-            })
-        end
+            end
+        })
+    end
+end
+
+local function createConfigBossZones()
+    for groupName, zone in pairs(sharedConfig.menus) do
+        createZones(groupName, zone.group, zone.coords, zone.size, zone.rotation)
     end
 end
 
+---Creates a boss zone for the specified group
+---@param groupName string Name of the group
+---@param groupType 'job'|'gang' Type of group
+---@param coords vector3 Coordinates of the zone
+---@param size? vector3 uses vec3(1.5, 1.5, 1.5) if not set
+---@param rotation? number uses 0.0 if not set
+local function createBossZones(groupName, groupType, coords, size, rotation)
```

Comment:
Consider using a table for the parameters. I think the table pattern provides a more readable function call and also allows us to more easily introduce changes in the future without breaking the API